README.md

```diff
@@ -42,7 +42,7 @@ Want to become a sponsor? Learn [what we do with sponsors' money](https://www.as
 
 ### Platinum
 
-<a href="https://solace.com" target="_blank">
+<a href="https://solace.com/?utm_source=github&utm_medium=onlinereferral&utm_campaign=asyncapi_sponsorship" target="_blank">
   <img src="./assets/solace.png" alt="Solace logo" height="70">
```

Comment:
do you want to also maybe suggest a better `altText`? 125char limit

I know one sponsor provided one explicitly because of SEO - I do not know if it improves SEO, but yeah, since you have PR opened, maybe you want to update it?

Reply:
good idea, I can add that. Unsure if it helps though!